packaging/tests/test_package.py

```diff
@@ -23,7 +23,8 @@ def test_package_main(self):
             path_to_test_file.unlink()
 
 
-        os.system("python -m package.package " + str(files_directory) + " --dest " + str(expected_dest_directory))
+        os.system("py -3 -m package.package " + str(files_directory) + " --package-only " + " --dest " + str(expected_dest_directory) )
```

Comment:
I think changing `python` to `py -3` will make this only work on Windows. The dev workflow for `packaging` specifies the use of a venv, IIRC. If the developer is using a venv it should be safe to just call `python` and get python3.7.x